chain/network/src/routing/edge.rs

```diff
@@ -283,18 +283,22 @@ impl SimpleEdge {
         self.nonce
     }
 
-    pub fn edge_type(&self) -> EdgeType {
+    pub fn edge_state(&self) -> EdgeState {
         if self.nonce % 2 == 1 {
-            EdgeType::Added
+            EdgeState::Active
         } else {
-            EdgeType::Removed
+            EdgeState::Removed
         }
     }
 }
 
-/// Status of the edge
+/// State of a given edge.
+/// Every edge starts in `Active` state. It can be removed and go to `Removed` state, and then
+/// added back to go to `Active` state, etc.
 #[derive(BorshSerialize, BorshDeserialize, Clone, PartialEq, Eq, Debug, Hash)]
-pub enum EdgeType {
-    Added,
+pub enum EdgeState {
+    /// `Edge` is `Active` if there is an active connection between two peers on the network.
+    Active,
+    /// `Edge` is in `Removed` state if it was previously in `Active` state, but has been removed.
```

Comment:
AFAIK node is removed only if we get an 'explicit' Remove request that is signed by one of the peers right?

I'd also mention that:

if we didn't 'see' the connection for a while, it will still be 'active' - but we might remove it from the memory.

Reply:
Thanks, I added extra explanation
```
   /// A signature of one of the peers is requires, otherwise the edge will stay active.
    /// Though, it may be removed  from memory if both peers become unreachable.
```